### PR TITLE
give consul same rule yaml names as kube bookinfo demo

### PR DESCRIPTION
**Pull request by GregHanson**
**Release note**:

```release-note
<PLEASE PUT RELEASE-NOTE HERE, PUT NONE if NO RELEASE-NOTE>
NONE
```

### CODE REVIEW

File: samples/bookinfo/consul/README.md [hunk 1]
```diff
@@ -28,9 +25,7 @@ The ingress controller is still under construction, routing functionalities can
 First step is to configure kubectl to use the apiserver created in the steps below:
```
Comment (ijsnellf) [inline samples/bookinfo/consul/README.md]: change `kubectl` to `istioctl`?

**Comment by ijsnellf** [comment]
here or in another PR you need to add `docker-compose` and `docker` as prereqs.

**Comment by ZackButcher** [comment]
LGTM

**Pull request merged**